code/backend/batch/AddURLEmbeddings.py

```diff
@@ -1,24 +1,26 @@
 import logging
 import traceback
 import azure.functions as func
-import sys
 
 from utilities.helpers.EnvHelper import EnvHelper
 from utilities.helpers.DocumentProcessorHelper import DocumentProcessor
 from utilities.helpers.ConfigHelper import ConfigHelper
 
-sys.path.append("..")
 
 bp_add_url_embeddings = func.Blueprint()
-env_helper: EnvHelper = EnvHelper()
-
-logger = logging.getLogger(__name__)
-logger.setLevel(env_helper.LOGLEVEL)
 
 
 @bp_add_url_embeddings.route(route="AddURLEmbeddings")
 def add_url_embeddings(req: func.HttpRequest) -> func.HttpResponse:
+    return do_add_url_embeddings(req)
+
+
+def do_add_url_embeddings(req: func.HttpRequest) -> func.HttpResponse:
+    logger = logging.getLogger(__name__)
```

Comment:
Is it possible to still leave the configuration of the logger where it was before?

Reply:
I think I can move create the logger itself outside, but to set the log level it now depends on the EnvHelper which needs to be mocked

Reply:
Oh I see

I think we should get it from `os.environ` without using the helper, like here

https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/blob/f39308437acbcbaa51f0ab57ed9ba8e34d695893/code/app.py#L6

Reply:
Tbh I'm not a fan of this, imo it would be better if the EnvHelper didn't come with so many side effects, but that's beside the scope of this PR :)

Reply:
Same tbh